### PR TITLE
Cu 85ztre9hq 1 landing screen nirwan rai

### PR DESCRIPTION
**Pull request by nirwan77**
made home page

### CODE REVIEW

**Comment by rubek-joshi** [comment]
Task linked: [CU-85ztre9hq 1. Landing Screen](https://app.clickup.com/t/85ztre9hq)

File: src/screens/Home/index.module.scss [hunk 1]
```diff
@@ -0,0 +1,138 @@
+@use "main";
+
+.logo{
+	@media only screen and (max-width: 600px) {
```
Comment (prakash-2021) [inline src/screens/Home/index.module.scss:4]: change it to global variable

File: src/screens/Home/index.module.scss [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    line-height: 150%; 
+    letter-spacing: -0.01em;
+
+	@media only screen and (max-width: 600px) {
+		text-align: center;
+		font-size: 18px;
+		font-style: normal;
+		font-weight: 500;
+		line-height: 150%; /* 27px */
+		letter-spacing: -0.18px;
```
Comment (prakash-2021) [inline src/screens/Home/index.module.scss:25]: should be in em

File: src/screens/Home/index.module.scss [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+    line-height: 100%; 
+    letter-spacing: -0.03em;
+
+
+	@media only screen and (max-width: 600px) {
+		font-size: 48px;
+		font-style: normal;
+		font-weight: 600;
+		line-height: 110%; /* 52.8px */
+		letter-spacing: -1.44px;
```
Comment (prakash-2021) [inline src/screens/Home/index.module.scss:43]: make letter spacing em for all

File: src/screens/Home/index.tsx [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+            </div>
+
+            <div className="flex align-center gap-4 pointer-events-auto">
+              <span className="text-base tracking-[0.01rem] leading-[140%] font-medium lg:ca-body--sm">
+                Share
+              </span>
+              <div className="flex gap-6 lg:gap-3">
+                <a className="cursor-pointer" href="https://www.google.com">
+                  <BiLink color={"#470FF4"} size={24} />
+                </a>
```
Comment (prakash-2021) [inline src/screens/Home/index.tsx:181]: spacingsss

File: src/styles/_layouts.scss [hunk 5]
```diff
@@ -10,3 +10,9 @@
   // background-color: variables.$yellow;
   // background: variables.$yellow;
 }
+
+.ca-gameOver {
+  // background-image: url("assets/backgroundPattern.svg");
```
Comment (prakash-2021) [inline src/styles/_layouts.scss:15]: remove unnecessary comments

**Review by prakash-2021 — CHANGES_REQUESTED** [review]

**Review by rubek-joshi — CHANGES_REQUESTED** [review]

File: src/screens/GameOver/index.tsx [hunk 6]
```diff
@@ -24,37 +24,49 @@ export const GameOver = () => {
...
           <div className={styles.innerContainer}>
             <img className={styles.image} src={trophy} alt={trophy} />
-            <div className="flex flex-col gap-10">
+
+            <div className="flex flex-col ca-gap--40">
               <div className="flex flex-col">
-                <div className="ca-heading--three">Appreciate your effort</div>
-                <div className="ca-body--sm">
+                <div className="ca-mb--8">
+                  <div className="ca-heading--three ">
```
Comment (prakash-2021) [inline src/screens/GameOver/index.tsx:31]: remove blank space

File: src/screens/Landing/index.module.scss [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+	margin: 0 auto;
+    padding-right: 1.25rem/* 20px */;
+    padding-left: 1.25rem/* 20px */;
+}
+
+
+.logo{
+	height: main.em(40);
+	width: auto;
+	// @media only screen and (max-width: main.$breakpoint-mobile) {
```
Comment (prakash-2021) [inline src/screens/Landing/index.module.scss:17]: remove comment if you need later write it in comment

File: src/screens/Landing/index.module.scss [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+    line-height: 150%; 
+    letter-spacing: -0.01em;
+
+	@media only screen and (max-width: main.$breakpoint-mobile) {
+		text-align: center;
+		font-size: main.em(18);
+		font-style: normal;
+		font-weight: 500;
+		line-height: 150%; /* 27px */
+		letter-spacing: -0.18px;
```
Comment (prakash-2021) [inline src/screens/Landing/index.module.scss]: letter spacing MUST be in em

File: src/screens/Landing/index.module.scss [hunk 9]
```diff
@@ -0,0 +1,132 @@
...
+	  }
+}
+
+.arrow{
+	@apply select-none;
+
```
Comment (prakash-2021) [inline src/screens/Landing/index.module.scss:43]: spacings ghin lagdo

File: src/screens/Landing/index.tsx [hunk 10]
```diff
@@ -1,16 +1,212 @@
...
+                  the disease
+                </p>
+              </div>
+
+              <div className="flex align-center ca-gap--16 pointer-events-auto select-none">
+                <span className="ca-body--sm">Share</span>
+                <div className="flex ca-gap--12">
+                  <a className="cursor-pointer" href="https://www.google.com">
+                    <BiLink className={styles.social} />
+                  </a>
```
Comment (prakash-2021) [inline src/screens/Landing/index.tsx:183]: spacings

File: src/screens/Quiz/Result.tsx [hunk 11]
```diff
@@ -2,47 +2,55 @@ import Correct from "assets/resultEmoji/right.png";
...
+              </p>
             </div>
 
-            <div className="ca-misc--link">
-              <a target="_blank" href={currentQuestion.source} rel="noreferrer">
-                Learn more
-              </a>
-            </div>
+            <a
+              className=" ca-misc--link "
```
Comment (prakash-2021) [inline src/screens/Quiz/Result.tsx:47]: blank space

File: src/screens/Quiz/components/GameActions.tsx [hunk 12]
```diff
@@ -11,24 +11,18 @@ export const GameActions = () => {
...
-          <IconButton
-            icon={<BiX size={64} />}
-            onClick={() => handleAnswer(false)}
-          />
-          <IconButton
-            icon={<BiCheck size={64} />}
-            onClick={() => handleAnswer(true)}
-          />
+        <div className="flex justify-center ca-mb--96 ca-gap--64">
+          <IconButton icon={<BiX />} onClick={() => handleAnswer(false)} />
```
Comment (prakash-2021) [inline src/screens/Quiz/components/GameActions.tsx:15]: use curry function if possible

**Review by prakash-2021 — CHANGES_REQUESTED** [review]

**Review by rubek-joshi — APPROVED** [review]

**Review by prakash-2021 — APPROVED** [review]

**Pull request merged**